lib/table.py

```diff
@@ -1,10 +1,15 @@
+from future import standard_library
+standard_library.install_aliases()
+from builtins import next
+from builtins import object
 import csv
-import StringIO
+import io
 
 from retriever.lib.cleanup import *
+from functools import reduce
```

Comment:
Imports should be grouped and ordered following https://www.python.org/dev/peps/pep-0008/#imports, so `functools` should be imported before the local retriever imports.